apps/nestjs-indexer/src/services/sell-token/sell-token.service.ts

```diff
@@ -40,26 +37,19 @@ export class SellTokenService {
 
       let price = tokenLaunchRecord?.price ?? 0;
 
-      let coinAmount = Number(data.coinAmount);
-      let quoteAmount = Number(data.amount);
-
-      // const calculatedQuoteAmount = Number(data.amount) * Number(price);
-      const calculatedQuoteAmount = Number(data.quoteAmount);
-
-      const effectiveQuoteAmount =
-        calculatedQuoteAmount;
-      // calculatedQuoteAmount - Number(data?.protocolFee);
+      const calculatedQuoteAmount = Number(data.amount) * Number(price);
```

Comment:
Wrong use of the event

Reply:
And not need to multiply per price, amount and quoteAmount is the amount event in the sellEvent